### PR TITLE
added chart, refactored server

### PR DESCRIPTION
**Pull request by ops1ops**

### CODE REVIEW

File: src/client/components/ItemPage/ItemPage.jsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        // TODO: handle error
+      }
+    };
+
+    handleItemFetch();
+  }, []);
+
+  return (
+    <div>
+      <canvas id="chard" ref={chartRef} />
```
Comment (dexter0175) [inline src/client/components/ItemPage/ItemPage.jsx]: chart?

File: src/client/components/RegisterPage/RegisterForm/RegisterForm.jsx [hunk 2]
```diff
@@ -49,25 +49,25 @@ const RegisterForm = () => {
...
   const handleRegister = useCallback(async (e) => {
     e.preventDefault();
-    const isValid = email
+    const areAllFieldsFilled = email
       && login
       && password
       && passwordConfirmation
       && password === passwordConfirmation;
 
-    if (isValid) {
```
Comment (dexter0175) [inline src/client/components/RegisterPage/RegisterForm/RegisterForm.jsx]: I think, isValid is more readable

File: src/client/components/ItemPage/ItemPage.jsx (hunk 1 above)
Comment (dexter0175) [reply src/client/components/ItemPage/ItemPage.jsx]: Maybe, we can somehow export History  chart into standalone component to later reuse it on other pages?

Comment (ops1ops) [reply src/client/components/ItemPage/ItemPage.jsx]: maybe but not in this PR

**Pull request merged**